source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -3,13 +3,7 @@
 Use VirtualBox\*
```

Comment:
Change the title from "Use VirtualBox\*" to "Run pre-configured Clear Linux as a VirtualBox guest OS"